server/mqtt.go

```diff
@@ -1378,6 +1486,32 @@ func (as *mqttAccountSessionManager) processSessionPersist(_ *subscription, pc *
 	sess.mu.Unlock()
 }
 
+func (as *mqttAccountSessionManager) sessPersistProcessing(closeCh chan struct{}) {
+	as.mu.RLock()
+	sp := &as.sp
+	quitCh := as.jsa.quitCh
```

Comment:
Should we guard jsa being nil?

Reply:
Never set to nil after being created. That's my MO..